conf/machine/include/imx-base.inc

```diff
@@ -376,6 +381,8 @@ KERNEL_IMAGETYPE_aarch64 = "Image"
 
 MACHINE_FEATURES = "usbgadget usbhost vfat alsa touchscreen"
 
+MACHINE_FEATURES_append_mx8m = " jailhouse"
```

Comment:
I was thinking about this and I believe we ought to use another mechanism for this. Please consider:

```
SOC_MACHINE_FEATURES_BACKFILL_mx8m = "jailhouse"
MACHINE_FEATURES_BACKFILL += "${SOC_MACHINE_FEATURES_BACKFILL}"
```

Reply:
Okay, I'm gonna test this.
So `MACHINE_FEATURES` will become an append of `MACHINE_FEATURES_BACKFILL`.